internal/core/command/get_test.go

```diff
@@ -108,9 +87,38 @@ func TestNewGetCommandNoCorrelationIDInContext(t *testing.T) {
 
 func TestNewGetCommandInvalidBaseUrl(t *testing.T) {
 	device := testDevice
+	req := newRequestForGetWithContentType(clients.ContentTypeCBOR)
 	device.Service.Addressable.Address = "!@#$"
-	_, err := NewGetCommand(device, testCommand, "", context.Background(), nil, logger.NewMockClient())
+	_, err := NewGetCommand(device, testCommand, context.Background(), nil, logger.NewMockClient(), req)
 	if err == nil {
 		t.Errorf("The invalid URL error was not properly propagated to the caller")
 	}
 }
+
+func TestNewGetCommandCborContentType(t *testing.T) {
```

Comment:
Can this test and `TestNewGetCommandJsonContentType` be condensed into a single test table since there is some code duplication? Also, can we create negative tests to ensure we are not passing  unexpected headers? 

This also applies to the tests in `put_test.go`